lib/inc/internal/util/agent.hpp

```diff
@@ -16,9 +16,20 @@ namespace facter { namespace util { namespace agent {
      * @param exe the name of the executable
      * @return the path to the executable
      */
-    inline std::string which(const std::string& exe) {
+inline std::string which(const std::string& exe) {
```

Comment:
indentation

Reply:
I have discarded all modification for this file. You are reviewing the version a02a661 which is three years old.